packages/react-avatar/src/components/Avatar/useAvatarStyles.ts

```diff
@@ -336,9 +336,7 @@ const useColorStyles = makeStyles({
 });
 
 export const useAvatarStyles = (state: AvatarState): AvatarState => {
-  const { size, shape, active, activeAppearance } = state;
-  // 'colorful' should have been replaced with a color name by useAvatar, but if not default to darkRed
-  const color = state.color === 'colorful' ? 'darkRed' : state.color;
```

Comment:
Did we just remove this logic?

Reply:
I see now that this should be ensured in the `useAvatar` hook so there is no need for this anymore.